src/ajax/core.cljc

```diff
@@ -541,17 +541,15 @@
     (keyword-response-format-element format format-params)))
 
 (defn print-response [response]
-  (println "CLJS-AJAX response:  " response))
+  (println "CLJS-AJAX response:" response))
 
 (def default-handler (atom print-response))
 
 (defn print-error-response [response]
-  #? (:clj (println "CLJS-AJAX ERROR: " response)
-      :cljs (if-let [c js/window.console]
-          (.error c response)
-          (if-let [w js/Window]
-            (.alert w (str response))
-            (println "CLJS-AJAX ERROR: " response)))))
+  #? (:clj  (println "CLJS-AJAX ERROR:" response)
+      :cljs (cond js/console (.error js/console response)
```

Comment:
Go ahead, I'll hold off pulling anything. :)

On Sunday, 5 June 2016, Cameron Desautels <notifications@github.com> wrote:

> In src/ajax/core.cljc
> <https://github.com/JulianBirch/cljs-ajax/pull/150#discussion_r65821202>:
>
> >
> >  (def default-handler (atom print-response))
> >
> >  (defn print-error-response [response]
> > -  #? (:clj (println "CLJS-AJAX ERROR: " response)
> > -      :cljs (if-let [c js/window.console]
> > -          (.error c response)
> > -          (if-let [w js/Window]
> > -            (.alert w (str response))
> > -            (println "CLJS-AJAX ERROR: " response)))))
> > +  #? (:clj  (println "CLJS-AJAX ERROR:" response)
> > +      :cljs (cond js/console (.error js/console response)
>
> That said, it does raise the point that whatever is being checked, exists?
> should probably be used. I'll make the change tomorrow if no one beats me
> to it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JulianBirch/cljs-ajax/pull/150/files/b56ccb97943654b6e1f359a2302633eaee9bc375#r65821202>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFIk190etlQi4IVZ2dQcEP-nyDPxhzyks5qIwtXgaJpZM4IuZHu>
> .
>


-- 
Sent from an iPhone, please excuse brevity and typos.